ETT-Backend.Tests/Services/EmployeeServiceTest.cs

```diff
@@ -0,0 +1,24 @@
+using Xunit;
+
+namespace ETT_Backend.Services.Test
+{
+  public class EmployeeServiceTest
+  {
+    [Fact]
+    public void RetrieveEmployeeHoursvalidEmail() 
+    {
+      var empService = new EmployeeService();
+      var hoursResponse = empService.RetrieveEmployeeMetrics("cmason@callibrity.com");
+      Assert.Equal("0919", hoursResponse.EmployeeId);
+      Assert.Equal(20, hoursResponse.Billable.CurrentHours);
+    }
+
+    [Fact]
+    public void RetrieveEmployeeHoursInvalidEmail() 
+    {
+      var empService = new EmployeeService();
+      var hoursResponse = empService.RetrieveEmployeeMetrics("test@doesntexist.com");
+      Assert.Equal(null, hoursResponse);
```

Comment:
Github really doesn't like this line... I guess use Assert.Null( ) ?

Reply:
done!